src/GoodsList.tsx

```diff
@@ -7,10 +7,14 @@ type Props = {
 
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
-    {goods.map(good => (
-      <li key={good.id} data-cy="good">
-        {good.name}
-      </li>
-    ))}
+    {goods.map(good => {
+      const { id, color, name } = good;
+
+      return (
+        <li key={id} data-cy="good" style={{ color: `${color}` }}>
```

Comment:
what typeof color?
what if pass like that?
```suggestion
        <li key={id} data-cy="good" style={{ color }}>
```